src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -33,26 +33,30 @@
     <UseDebugLibraries>true</UseDebugLibraries>
     <PlatformToolset>v142</PlatformToolset>
     <CharacterSet>Unicode</CharacterSet>
+    <VCToolsVersion />
```

Comment:
Lost of repetition here. Can these be in the "Globals" property group or an unconditional "Configuration" property group?